<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>741</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Persons">
     <attribute name="title">
      <string>Persons</string>
     </attribute>
     <widget class="QTableWidget" name="tablePersons">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>721</width>
        <height>381</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gender</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year of birth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year of death</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Computer</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="search_person">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addPersonButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Add person</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Computers">
     <attribute name="title">
      <string>Computers</string>
     </attribute>
     <widget class="QTableWidget" name="computerTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>721</width>
        <height>381</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>It was made</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Creator</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="addComputerButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Add computer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
